### ISSUE TITLE
Height of 'list of sub-regions' when region is open

### ISSUE THREAD

**Issue by bradleyflood** [TheRevivalFellowship/Global-Website#7]
Should the list of sub-regions (i.e. Fellowships in Australia) be limited in height? So that that height is the same as the height of a fellowship detail view when opened. This would fix the jerky reflow when a list is collapsed but means the user must scroll within a smaller area -- perhaps not the best UX.

**Comment by danielmee** [comment]
Would it be possible to have sticky headers and fellowship details? So if you open an area (say Australia) and you scroll down, when the Blue and Dark Grey bands get to the top they'lll fix there. When they click on the assembly, it will appear under those bands. When the close button is pressed for the assembly the revious scroll position should be still where it was. When the 'x' is pressed for the location it may need a scrollto event to the correct position.
I noticed that currently when clicking on an assembly the list of other assemblies disappears - this may be causing some of the jumpy problems. Would it be possible to think of the assembly as an overlay on top of the listing?

**Comment by bradleyflood** [comment]
Yes sticky is possible and I think the best experience. But it takes more time. For now I say we fix the height of the detail and list view, that's a quick fix, and then I will address the sticky at a later date post launch.

TODO me:
1. Fix height and jerkiness issues of fellowships.
2. Sticky header of fellowship list etc.
